parser-typechecker/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -932,17 +932,25 @@ viewPatch = InputPattern "view.patch" [] [(Required, patchArg)]
    )
 
 link :: InputPattern
-link = InputPattern "link" []
-  [(Required, exactDefinitionQueryArg),
-   (Required, exactDefinitionQueryArg) ]
-  "`link src dest` creates a link from `src` to `dest`. Use `links src` or `links src <type>` to view outgoing links, and `unlink src dest` to remove a link."
+link = InputPattern
+  "link"
+  []
+  [(Required, exactDefinitionQueryArg), (OnePlus, exactDefinitionQueryArg)]
+  (fromString $ concat
+    [ "`link dest src` creates a link to `dest` from `src`. "
+    , "Use `links src` or `links src <type>` to view outgoing links, "
+    , "and `unlink dest src` to remove a link. The `src` can be either the "
+    , "name of a term or type, multiple such names, or a range like `1-4` "
+    , "for a range of definitions listed by a prior `find` command."
+    ]
```

Comment:
I get confused (especially with the earlier wording) about what `dest` and `src` are.  What do you think about `defn` and `metadata...`?